prisma/schema/application.prisma

```diff
@@ -19,29 +14,52 @@ enum NextAction {
   CANDIDATE_INFORMED
 }
 
+enum QualificationType16 {
+  GCSE
+  INDIAN_X
+  MALAYSIAN
+  SINGAPORE
+  OTHER
+}
+
+enum QualificationType18 {
+  A_LEVEL
+  INDIAN_XII
+  MALAYSIAN
+  SINGAPORE
+  ROMANIAN
+  OTHER
+}
```

Comment:
Should we still call these GcseQualification and ALevelsQualification? 
The name Type16 and Type18 still comes across as misleading or meaningless without the full context.

Reply:
Renamed in [d57f8ed](https://github.com/imperial/ug-admissions/pull/6/commits/d57f8ed7fa67621ad98eb1d8323a9f789356ecdc)